react/official/tabs/src/App.tsx

```diff
@@ -56,15 +56,15 @@ const App: React.FC = () => (
         </IonRouterOutlet>
         <IonTabBar slot="bottom">
           <IonTabButton tab="tab1" href="/tab1">
-            <IonIcon icon={triangle} />
+            <IonIcon aria-hidden={true} icon={triangle} />
             <IonLabel>Tab 1</IonLabel>
           </IonTabButton>
           <IonTabButton tab="tab2" href="/tab2">
-            <IonIcon icon={ellipse} />
+            <IonIcon aria-hidden={true} icon={ellipse} />
```

Comment:
```suggestion
            <IonIcon aria-hidden="true" icon={ellipse} />
```

`aria-hidden` is an attribute not a property, so it accepts the string `"true"` not the boolean.